Interfaces/about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../css/main.css" />
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="../css/about.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <title>About</title>
+  </head>
+  <body>
+    <header class="header-container">
+      <div class="circle"></div>
+      <nav>
+        <ul class="first-navbar">
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li><a>English</a></li>
+          <li><a>My account</a></li>
+          <li><a>logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <a href="../index.html"
+          ><img
+            class="brand-logo"
+            src="../img/logo.png"
+            alt="Brand-name-and-logo"
+        /></a>
+        <ul class="nav-list-items">
+          <li class="list-item"><a href="../index.html">Home</a></li>
+          <li class="list-item">
+            <a href="../Interfaces/About.html">About</a>
+          </li>
```

Comment:
- [ ] Kindly make sure you remove this **option** from your navigation options as once it is clicked it shows an error page.

![Screenshot from 2022-09-22 12-16-16](https://user-images.githubusercontent.com/83042742/191721567-45b6f8e1-2aad-40fc-adf9-961950e1ea0b.png)